data/lipophilicity/meta.yaml

```diff
@@ -0,0 +1,37 @@
+name: lipophilicity
+description: Experimental results of octanol/water distribution coefficient (logD
+  at pH 7.4).
+targets:
+- id: exp
+  description: experimental results of octanol/water distribution coefficient (logD
+    at pH 7.4)
+  units: logD
+  type: continuous
+  names:
+  - octanol/water distribution coefficient (logD at pH 7.4)
+  - octanol/water distribution coefficient
+identifiers:
+- id: SMILES
+  type: SMILES
+  description: SMILES
+license: CC BY-SA 3.0
+links:
+- url: https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/Lipophilicity.csv
+  description: original dataset
+- url: https://github.com/cheminfo/molecule-features/blob/main/data/lipophilicity/meta.yaml
+  description: original meta data
+- url: https://deepchem.readthedocs.io/en/latest/api_reference/moleculenet.html#lipo-datasets
+  description: original dataset link from moleculenet
+- url: https://www.ebi.ac.uk/chembl/document_report_card/CHEMBL3301361/
+  description: original report card
+- url: https://chembl.gitbook.io/chembl-interface-documentation/about#data-licensing
+  description: original dataset license from chembl
+- url: https://creativecommons.org/licenses/by-sa/3.0/
+  description: used dataset license
+num_points: 4200
+url: https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/Lipophilicity.csv
```

Comment:
```suggestion
url: https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/Lipophilicity.csv
```

Reply:
or do you think it is better to keep it here. I'm fine with both as long as we have a way of easily finding the original dataset

Reply:
fixed

Reply:
also the reason for the strange yaml multiline behavior here was the special characters, after the removal it worked.